tests/tests_default.yml

```diff
@@ -2,6 +2,54 @@
 ---
 - name: Ensure that the role runs with default parameters
   hosts: all
-  gather_facts: false  # test that role works in this case
-  roles:
-    - linux-system-roles.template
+  gather_facts: true  # test that role works in this case
```

Comment:
The role must work with `gather_facts: false`, this is consistent across all system roles. It is required because some customers run roles with `gather_facts: false` to save time. See https://github.com/linux-system-roles/storage/pull/262 that implements this.

Reply:
For more info on tox, see https://github.com/linux-system-roles/tox-lsr#qemu-testing